README.md

```diff
@@ -356,6 +356,98 @@ saveAsync(obj1).onSuccessTask(new Continuation<ParseObject, Task<ParseObject>>()
 });
 ```
 
+## Cancelling Tasks
+
+To cancel a task create a `CancellationTokenSource` and pass the corresponding token to any methods that create a task you want to cancel, then call `cancel()` on the source. This will cancel any ongoing tasks that the token was supplied to.
+
+
+```java
+CancellationTokenSource cts = new CancellationTokenSource();
+
+Task<Integer> stringTask = getIntAsync(cts.getToken());
+
+cts.cancel();
+```
+
+To cancel an asynchronous call using a token you must first modify the method to accept a `CancellationToken` and use the `throwIfCancellationRequested()` and `isCancellationRequested()` methods to halt the operation once cancelled.
+
+```java
+/**
+ Gets an Integer asynchronously.
+ */
+public Task<Integer> getIntAsync(CancellationToken ct) {
+  // Create a new Task
+  Task<Integer>.CompletionSource tcs = Task.create();
+
+  new Thread() {
+    @Override
+    public void run() {
+      // Check if cancelled at start
+      if (ct.isCancellationRequested()) {
+        tcs.setCancelled();
+        ct.throwIfCancellationRequested();
```

Comment:
I think this sample code would crash right here due to an uncaught exception in a new thread? We probably want `return;` instead.